include/SKSE/Interfaces.h

```diff
@@ -374,19 +374,26 @@ namespace SKSE
 		constexpr void MinimumRequiredXSEVersion(REL::Version a_version) noexcept { xseMinimum = a_version.pack(); }
 		constexpr void PluginName(std::string_view a_plugin) noexcept { SetCharBuffer(a_plugin, std::span{ pluginName }); }
 		constexpr void PluginVersion(REL::Version a_version) noexcept { pluginVersion = a_version.pack(); }
+		constexpr void HasNoStructUse(bool a_value) noexcept { noStructUse = a_value; }
 		constexpr void UsesAddressLibrary(bool a_value) noexcept { addressLibrary = a_value; }
 		constexpr void UsesSigScanning(bool a_value) noexcept { sigScanning = a_value; }
+		constexpr void UsesStructsPost629(bool a_value) noexcept { structsPost629 = a_value; }
 
 		const std::uint32_t dataVersion{ kVersion };
 		std::uint32_t       pluginVersion = 0;
 		char                pluginName[256] = {};
 		char                author[256] = {};
 		char                supportEmail[256] = {};
```

Comment:
```suggestion
		char                supportEmail[252] = {};
```
He stole some bytes from here to make room for the new field
https://github.com/ianpatt/skse64/blob/7adb73617075a82bff6b3bce198bb6f1fc0174a2/skse64/PluginAPI.h#L327